acceptance/acceptance_suite_test.go

```diff
@@ -32,6 +32,27 @@ var nodeSuffix, nodeTmpDir string
 var serverURL string
 var registryMirrorName = "epinio-acceptance-registry-mirror"
 
+// Configuration Notes
+//
+// - skipCleanupPath: Path relative to the test directory.  IOW
```

Comment:
As far as I have seen, ginkgo always changes the working dir to the directory the test suite is in.

I'm not certain what this documentation tries to achieve: 

* If it's a manual on how to configure tests, it might be better to have it in `docs/running-tests.md`, drop all the variable names and mention the files and env vars directly.

* If this is meant to document the test_suite code, it would be nice to move the comments onto the corresponding variables, so they show up when hovering over the variable name and in docs.
Also starting the comment with the name of the thing [seems to help with go doc](https://golang.org/doc/effective_go#commentary): https://pkg.go.dev/github.com/epinio/epinio?utm_source=godoc

Reply:
I wanted some documentation which helps me as an epinio dev to remember how I can configure our :cat2: .
So I guess the first point. Although the second point is a good idea also IMHO.

I see no `docs/running-tests.md` in the current `main` commit.
Guessing now that you are proposing to create such a doc file, right ?

Will let this bake a bit more in the back of my mind, focus now is the auth, and this PR is not an urgent thing/

Reply:
Yep. Pushed reworked commit